NZazu/Fields/PropertyExtensions.cs

```diff
@@ -93,8 +92,8 @@ private static object GetConvertedValue(string propValue, PropertyInfo propInfo)
             }
             catch (Exception ex)
             {
-                Trace.TraceError("Could not convert the value \"{0}\" for property type \"{1}\" of prop \"{2}\": {3}", propValue, propInfo?.PropertyType.Name, propInfo?.Name, ex);
-                return null;
+                var message = $"Could not convert the value \"{propValue}\" for property type \"{propInfo?.PropertyType.Name}\" of prop \"{propInfo?.Name}\"";
```

Comment:
Losing original exception here. Is this what you want?

Reply:
This is not specifically what i want, no. I just thought that it would be helpful for users (in this case: developer / maintainer) to see which Control raises an error, when it's not able to apply certain settings.

An Exception will be raised, i.e. when the Setting `Width` is set to an empty string ("") inside the template (this is where this problem was brought up, because we had to search for the according control).

What would be a better approach?

Reply:
Probably the templates are getting too complicated? This would indicate that the user is trying to accomplish too much with a single template.

I can imagine that in working with e.g. NZazuFiddlet to try/explore template designs, the workflow should provide 1st-class feedback.

Reply:
Ah, forget it. I overlooked that the original exception is passed in as inner exception, so context is preserved. Should be ok